src/strategies_and_params/replicastrategy.jl

```diff
@@ -100,22 +107,26 @@ where
 is the (right) eigenvector of ``\\hat{G}`` and ``| \\psi \\rangle`` is an eigenvector of
 ``\\hat{H}``.
 
-For a K-tuple of input operators ``(\\hat{A}_1, ..., \\hat{A}_K)``, overlaps of
+For an m-tuple of input operators ``(\\hat{A}_1, ..., \\hat{A}_m)``, overlaps of
```

Comment:
```suggestion
For an ``m``-tuple of input operators ``(\\hat{A}_1, ..., \\hat{A}_m)``, overlaps of
```
this will render m in LaTeX right?